lib/template.js

```diff
@@ -28,26 +30,29 @@ const template = (string, obj, customReplacers) => {
 function validateReplacers({ app, context, replacers }) {
   return replacers
     .map(replacer => {
-      if (replacer.flags === undefined || replacer.flags === null) {
-        replacer.flags = 'g'
-      }
-      return replacer
-    })
-    .filter(replacer => {
       try {
-        new RegExp(replacer.search, replacers.flags)
-        return true
+        if (replacer.search.startsWith('/')) {
```

Comment:
what if it is a part of a URL :laughing:

Reply:
it'll silently fail?

Reply:
> it'll silently fail?

exactly my point and they would have to do `/\/my-url/g` :)

Reply:
they sure will

Reply:
only solution i see is to make 2 new fields, string and regex, and be explicit

Reply:
For most simple regex which I imagine will be most, they can get away with stuff like `JENKINS-(\d+)` or similar hence my reluctance to add more explicit behavior

Reply:
okay, changed it to be explicit

Reply:
I appear to get notifications late.

I'm indifferent.

Just tell me the behavior you want